src/Mbstring/mb_convert_variables.70400.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\Polyfill\Mbstring as p;
+
+if (!function_exists('mb_convert_variables')) {
+    /**
+     * Convert character code in variable(s)
+     */
+    function mb_convert_variables($to_encoding, $from_encoding, &$var, &...$vars)
```

Comment:
is this the exact signature as reported by reflection since PHP 74?
let's put these extra files in the Resources folder.